pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -130,7 +131,7 @@ style_min: false
     });
     tutorialExplorerManager.renderToElement(document.getElementById('tutorials'));
 
-    var studentImage = '/images/learn/students.png';
+    var studentImage = '/images/learn/hoc2018_paint.png'; // hoc2018 override
```

Comment:
I appreciate all the hoc2018 comments - will make it easier to switch styling again when the time comes!

Reply:
I'm already second-guessing some of this PR, and thinking I could have broken these out into even more self-contained components with their styling baked in, but this will do for now!